src/serlo-editor/plugins/text/hooks/use-suggestions.tsx

```diff
@@ -230,13 +230,24 @@ function filterPlugins(
   const search = text.replace('/', '').toLowerCase()
   if (!search.length) return plugins
 
-  const startingWithSearchString = plugins.filter(({ title }) => {
-    return title.toLowerCase()?.startsWith(search)
+  const filterResults = new Set<SuggestionOption>()
+
+  // title or pluginType start with search string
+  plugins.forEach((entry) => {
+    if (
+      entry.title.toLowerCase()?.startsWith(search) ||
+      entry.pluginType.startsWith(search)
+    ) {
+      filterResults.add(entry)
+    }
   })
-  const containingSearchString = plugins.filter(({ title }) => {
-    const value = title?.toLowerCase()
-    return value?.includes(search) && !value?.startsWith(search)
+
+  // title includes search string
+  plugins.forEach((entry) => {
+    if (entry.title.toLowerCase()?.includes(search)) {
```

Comment:
Why do you need `?` after `toLowerCase` here and on line 238? 😄 It's supposed to always return a string

Reply:
![giphy](https://github.com/serlo/frontend/assets/13198821/0bb2bfbf-dbf1-4d88-bf7c-187a6e570db7)